app/android/app/src/main/AndroidManifest.xml

```diff
@@ -40,4 +40,21 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
+    <!-- Provide required visibility configuration for API level 30 and above -->
```

Comment:
Can you elaborate what these are for? I'd rather we keep the number of permissions we ask from your users to an absolute minimum...

Reply:
Above android 11 OS, this are the queries tabs we need to on the android manifest file to support default link actions.
Ref: https://pub.dev/packages/url_launcher

Reply:
I see... but then we should add the [same for `iOS`](https://pub.dev/packages/url_launcher#ios), too, no?

Reply:
On another note: we aren't actually checking whether the host can open it. In the utils we are checking whether we consider it an internal link and if not we send it forward ... We might want to reconsider that approach then...

Reply:
I have made the required changes. Thanks